app/screens/TalkDetailsScreen/TalkDetailsScreen.tsx

```diff
@@ -295,3 +342,35 @@ const $subtitle: TextStyle = {
 const $headingContainer: ViewStyle = {
   marginBottom: spacing.extraLarge,
 }
+
+const $assistantsContainer: ViewStyle = {
+  flexDirection: "row",
+  justifyContent: "space-around",
```

Comment:
I'm not sure if it's the `space-around` vs `space-evenly` (or some other value), it could just be my eyes on the screenshot.

It looks like to me that when there is more than 1 assistant, there is more space on the left side than the right.

Reply:
I believe that's my lack of taking a good screenshot 🤔